ui/raidboss/data/00-misc/test.ts

```diff
@@ -514,6 +542,14 @@ const triggerSet: TriggerSet<Data> = {
         'Super Tankbuster': '초강력 탱크버스터',
         'Pentacle Sac': 'Pentacle Sac',
         'Engage': '시작',
+        'Two': '둘',
+        '(?<! )Three': '셋',
+        'Four': '넷',
+        'Six': '여섯',
+        'Ten': '열',
+        'Fifteen': '십오',
+        'Force Jump Three': '세로 뛰어넘기',
+        'Invisible': '보이지 않는',
```

Comment:
Long time no see! 😄 

```suggestion
        'Two': '2',
        '(?<! )Three': '3',
        'Four': '4',
        'Six': '6',
        'Ten': '10',
        'Fifteen': '15',
        'Force Jump Three': '3으로 돌아가기',
        'Invisible': '타임라인 숨기기',
```

Using Arabic numerals is more natural. Interesting to recognize that Korean is the only one who uses number itself as a written text.

Reply:
That's fair.

And I have add another string that needs you to review again, sorry.

Reply:
`cactbot test combatant cast`

I don't understand what this means. Is this testing if recognizing skills which needs target works?

Reply:
This is a test for "whether cactbot could catch a spell casting game log". eg. Fire IV of Black Mage or something.

Reply:
By the way, I translate it in Japanese literally as "cactbot target cast test".

Reply:
I think it works in Korean just by transliterating that English one 🤣